matrix-sdk-android/src/main/java/org/matrix/android/sdk/internal/crypto/DefaultCryptoService.kt

```diff
@@ -524,21 +421,19 @@ internal class DefaultCryptoService @Inject constructor(
      * @param deviceId   the unique identifier for the device.
      */
     override fun setDeviceVerification(trustLevel: DeviceTrustLevel, userId: String, deviceId: String) {
-        setDeviceVerificationAction.handle(trustLevel, userId, deviceId)
+        // TODO
```

Comment:
rust SDK is not yet storing trust of devices?

Reply:
It does, but as verification isn't yet implemented, there wasn't a point in forwarding the info from the rust side, I also need to expose, on the rust side that is, if it's locally verified or cross signing verified, currently there's just a boolean.